### PR TITLE
Manual week blog post

### PR DESCRIPTION
**Pull request by couet**

### CODE REVIEW

**Comment by couet** [comment]
good point... I missed that.. let me reorganised

File: blog/_posts/2021-10-06-manual-week.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+_[Reference Guide](https://root.cern/doc/master/index.html)_. Finally we will drop
+completely the _Old User’s Guide_ and have an accurate, minimal but sufficiently complete
+_[Manual]({{ '/manual' | relative_url }})_.
+
+The new structure _[Manual]({{ '/manual' | relative_url }})_/_[Reference Guide](https://root.cern/doc/master/index.html)_
+will work in a more complementary, simpler and proactive way for updates either
+via Pull requests (on [web site repository](https://github.com/root-project/web/pulls) or
+[ROOT repository](https://github.com/root-project/root)) or Github issues
+(on [web site repository](https://github.com/root-project/web/pulls) or
+[ROOT repository](https://github.com/root-project/root/pulls)).
```
Comment (Axel-Naumann) [inline blog/_posts/2021-10-06-manual-week.md:43]: Maybe rephrase that to get our readers to interact? I think that was your plan? I.e. maybe:

This new manual allows *you* to contribute - for instance by letting us know when something is hard to understand (by opening an [issue](https://github.com/root-project/root/issues/new or whatever the link is)) or even by fixing it yourself: see the GitHub octocat at the bottom of each page!

And then conclude with something "warm":

We hope you enjoy the new manual, and that it's useful for today's new grad students!

File: blog/_posts/2021-10-06-manual-week.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - The _[Reference Guide](https://root.cern/doc/master/index.html)_
+  - The _[Manual]({{ '/manual' | relative_url }})_
+  - The _Old ROOT User’s Guide_
+
+The _[Reference Guide](https://root.cern/doc/master/index.html)_ and the
+_[Manual]({{ '/manual' | relative_url }})_ are the current valid sources of documentation.
+_[The Manual]({{ '/manual' | relative_url }})_ acts as a "User's Guide" helping users to find their way into the huge amount of documentation
+provided in the _[Reference Guide](https://root.cern/doc/master/index.html)_.
+The _Old User's Guide_ is outdated and not updated but, nevertheless it contains some valuable
+information we do not want to lose.
```
Comment (Axel-Naumann) [inline blog/_posts/2021-10-06-manual-week.md:28]:
```suggestion
The _Old User's Guide_ was outdated and not updated but nevertheless contained some valuable
information we did not want to lose.
```
(past tense to clarify that we don't think it's needed anymore after our "intervention")

File: blog/_posts/2021-10-06-manual-week.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+and create the new needed ones ([JSROOT]({{ '/manual/jsroot' | relative_url }}), [
+ROOT I/O]({{ '/manual/root_io' | relative_url }})...) which also
+led to updates in the _[Reference Guide](https://root.cern/doc/master/index.html)_.
+
+Another task was be to move all the valuable information now in the _Old User’s
+Guide_ to the relevant places: outdated parts were removed or
+updated in the _[Manual]({{ '/manual' | relative_url }})_ or in the
+_[Reference Guide](https://root.cern/doc/master/index.html)_. Finally we will drop
+completely the _Old User’s Guide_ and have an accurate and complete
+_[Manual]({{ '/manual' | relative_url }})_.
```
Comment (Axel-Naumann) [inline blog/_posts/2021-10-06-manual-week.md:42]:
```suggestion
_[Reference Guide](https://root.cern/doc/master/index.html)_. In the end (and after another, final round at the end of the year) we will drop
completely the _Old User’s Guide_ and instead have an accurate and complete
_[Manual]({{ '/manual' | relative_url }})_.
```
